apps/actors/lib/actors/actor/entity.ex

```diff
@@ -230,6 +230,41 @@ defmodule Actors.Actor.Entity do
       current_context: Context.new(state: current_state)
     )
     |> invoke_host(state)
+    |> case do
+      {:ok, response, state} -> {:reply, {:ok, response}, state}
+      {:error, :no_content, state} -> {:reply, {:error, :no_content}, state}
```

Comment:
I think it's good to put him to sleep in all cases of error (remembering that he will only sleep anyway if his run queue is empty)